styles.css

```diff
@@ -491,178 +488,178 @@ hr {
 }
 
 @media (min-width: 768px) {
-.container1 {
-  height: 916px;
- }
-
-nav {
-  height: 72px;
- }
-
-.logo {
-  margin-left: 2em;
- }
-
-.mobile-icons {
-  display: none;
- }
-
-.home-icons {
-  display: flex;
- }
-
-.P-Text {
-  align-content: center;
-  width: 63%;
- }
-
-.P-Text h1 {
-  font-size: 48px;
- }
-
-.P-Text p {
-  font-size: 20px;
- }
-
-.page-2-description {
-  width: 100%;
- }
-
-.project1 {
-  flex-direction: row;
-  column-gap: 1rem;
-  width: 90%;
- }
-
-.project2 {
-  flex-direction: row-reverse;
-  column-gap: 1rem;
-  width: 90%;
- }
-
-.project3 {
-  flex-direction: row;
-  column-gap: 1rem;
-  width: 90%;
- }
-
-.project4 {
-  flex-direction: row-reverse;
-  column-gap: 1rem;
-  width: 90%;
- }
-
-.web-icons{
-  width: 394px;
- }
-
-.pager-head {
-  font-size: 38px;
-  line-height: 52px;
- }
-
-.program-used {
-  padding: 20px;
- }
-
-.ul-details {
-  font-size: 12px;
- }
-
-.p-details {
-  font-size: 18px;
- }
-
-.p-ul-details {
-  width: 80%;
-  font-size: 16px;
-  line-height: 24px;
- }
-
-.about {
-  flex-direction: row;
-  height: 628px;
-  justify-content: center;
-  align-items: center;
-  padding: 0 8em 0 8em;
-  column-gap: 2em;
- }
-
-.user-tool {
-  width: 60%;
- }
-
-.user-tool-d {
-  width: 60%;
-  padding-top: 20px;
-  margin-bottom: 20px;
- }
-
-.me {
-  width: 70%;
- }
-
-.me p {
-  width: 447px;
-  padding-bottom: 15px;
- }
-
-.me h2 {
-  padding-bottom: 15px;
- }
-
-.tool-list {
-  display: flex;
-  flex-direction: row;
-  width: 100%;
-  padding-top: 38px;
- }
-
-.tool-list > li {
-  height: 122px;
-  flex-direction: column;
-  align-items: flex-start;
-  justify-content: center;
-  margin: 0 1em 1em 0;
-  width: 120px;
- }
-
-.skills {
-  padding-top: 20px;
- }
-
-.contact-form p {
-  width: 47%;
- }
-
-.respo-icons {
-  display: none;
- }
-
-.frameworks {
-  padding-top: 20px;
- }
-
-.ul-form {
-  list-style: none;
- }
-
-#name_form {
-  padding-left: 20px;
- }
-
-#email_form {
-  padding-left: 20px;
- }
-
-#message {
-  padding-left: 20px;
- }
-
-.main-icons {
-  display: block;
-  width: 55%;
-  height: auto;
-  margin: 10px 0 10px 10px;
- }
+  .container1 {
+    height: 916px;
+  }
+
+  nav {
+    height: 72px;
+  }
+
+  .logo {
+    margin-left: 2em;
+  }
+
+  .mobile-icons {
+    display: none;
+  }
+
+  .home-icons {
+    display: flex;
+  }
+
+  .P-Text {
+    align-content: center;
+    width: 63%;
+  }
+
+  .P-Text h1 {
+    font-size: 48px;
+  }
+
+  .P-Text p {
+    font-size: 20px;
+  }
+
+  .page-2-description {
+    width: 100%;
+  }
+
+  .project1 {
+    flex-direction: row;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .project2 {
+    flex-direction: row-reverse;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .project3 {
+    flex-direction: row;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .project4 {
+    flex-direction: row-reverse;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
```

Comment:
- [ ] From the actual design, the project cards are suppose to have the same dimensions, including height and width, however, I see that some of the cards have different dimensions as shown in the image below. 
- [ ] The project card is divided into two, half of it contains the project's picture and the other half contains the project's description, however, at certain screen widths of the desktop version, your project description covers more space than the project picture 
Kindly check and fix this issues to match the actual design and produce a more professional project. 


### Your design 

![Screenshot from 2023-02-25 16-54-15](https://user-images.githubusercontent.com/65233860/221369750-78f13fb6-e497-4c68-afa7-48356e9b4478.png)



### Your design 

![Screenshot from 2023-02-25 17-01-44](https://user-images.githubusercontent.com/65233860/221369794-c5da9b29-1a6e-4328-aa29-839489432112.png)



### Actual design 


![Screenshot from 2023-02-25 17-03-58](https://user-images.githubusercontent.com/65233860/221369879-f2e0edbd-c1f3-426c-abf3-97e4f7bf2009.png)